lambdas/tests/unit/services/test_dynamo_service_get_table.py

```diff
@@ -0,0 +1,52 @@
+from unittest.mock import MagicMock, patch
+
+import boto3
+import pytest
+from services.dynamo_service import DynamoDBService
+from tests.unit.helpers.data.dynamo_responses import MOCK_RESPONSE
+from botocore.exceptions import ClientError
+
+
+@pytest.fixture
+def mock_boto3_dynamo():
+    return MagicMock()
+
+
+@pytest.fixture
+def mock_dynamo_table():
+    return MagicMock()
```

Comment:
I also think that Abbas was concerned about us using overlapping test libraries and we were to avoid MagicMock in favour of Mocker, but I'm not sure on the status of that. I've always had to use with patch.object myself, not really a change just one to be aware of